### PR TITLE
add remove child and reset grid

### PR DESCRIPTION
**Pull request by Sawalhah**
I've added a new small button on each created child to remove
and added reset button to clear the grid

### CODE REVIEW

**Comment by lipis** [comment]
I think that's the best solution for removing divs :)

File: src/components/AppGrid.vue [hunk 1]
```diff
@@ -49,7 +49,9 @@
           :key="child"
           :class="'child' + i"
           :style="{ gridArea: child }"
-        ></div>
+        >
+          <button @click="removeChild(i)">(x)</button>
```
Comment (lipis) [inline src/components/AppGrid.vue]: just an `x` is enough I would say.. no need for `()`

File: src/components/AppGrid.vue [hunk 2]
```diff
@@ -114,6 +116,10 @@ export default {
 
         this.$store.commit("addChildren", childstring);
       }
+    },
+    removeChild(index) {
+      console.log(index);
```
Comment (lipis) [inline src/components/AppGrid.vue]: remove log :)

File: src/styles/main.scss [hunk 3]
```diff
@@ -73,8 +73,19 @@ button {
...
   &:hover {
     background: $teal;
     color: black;
   }
 }
+
+[type="reset"] {
+  color: $grey;
+  border: 1px solid $grey;
+
```
Comment (lipis) [inline src/styles/main.scss]:
```suggestion
width: 100%;
```

File: src/store.js [hunk 4]
```diff
@@ -42,13 +42,18 @@ export default new Vuex.Store({
       } else {
         let difference = newVal - oldVal;
         for (let i = 1; i <= difference; i++) {
-          state[payload.direction].push({ unit: "1fr" });
+          state[payload.direction].push({
+            unit: "1fr"
+          });
```
Comment (lipis) [inline src/store.js]: Changing the format shouldn't be part of this PR

File: src/store.js [hunk 5]
```diff
@@ -60,13 +65,18 @@ export default new Vuex.Store({
...
   }
 });
 
 //we start off with just a few rows and columns filled with 1fr units
 const createArr = (direction, arr) => {
   for (let i = 1; i <= direction; i++) {
-    arr.push({ unit: "1fr" });
+    arr.push({
+      unit: "1fr"
+    });
```
Comment (lipis) [inline src/store.js]: not related to this PR

File: src/styles/main.scss [hunk 6]
```diff
@@ -9,25 +10,24 @@ body {
...
+      ellipse cover,
+      #192d38 0%,
+      #211f2f 100%);
+  /* Chrome10-25,Safari5.1-6 */
+  background: radial-gradient(ellipse at center,
+      #192d38 0%,
+      #211f2f 100%);
+  /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#192d38', endColorstr='#211f2f', GradientType=1);
+  /* IE6-9 fallback on horizontal gradient */
```
Comment (lipis) [inline src/styles/main.scss]: not related to this PR

File: src/components/AppGrid.vue (hunk 1 above)
Comment (dasDaniel) [reply src/components/AppGrid.vue]: what about `&times;` ?

**Comment by equinusocio** [comment]
Is this the same of https://github.com/sdras/cssgridgenerator/pull/19

File: src/styles/main.scss (hunk 3 above)
Comment (sdras) [reply src/styles/main.scss]: No, I think it's better smaller.

**Comment by sdras** [comment]
Thanks @lipis for the good review too!

**Comment by Sawalhah** [comment]
> Of all of the solutions that people have PR'ed, I think this is the best one. Clean and easy to understand. Not overengineered. Nice work!

thanks :)
